modules/vba_documents/lib/vba_documents/upload_validator.rb

```diff
@@ -76,6 +76,7 @@ def validate_documents(parts, pdf_validator_options = VBADocuments::DocumentRequ
 
     def perfect_metadata(model, parts, timestamp)
       metadata = JSON.parse(parts['metadata'])
+      metadata['fileNumber'] = metadata['fileNumber']&.strip if metadata.key?('fileNumber')
```

Comment:
Since execution won't reach `perfect_metadata` if a `fileNumber` key is not provided and confirmed valid in the metadata (`validate_metadata` runs before `perfect_metadata`), `if metadata.key?('fileNumber')` and the safe navigation operator (`&.`) aren't needed and this can be simplified to:
```
metadata['fileNumber'] = metadata['fileNumber'].strip
```

Reply:
fixed